modules/mapper-extras/src/main/java/org/elasticsearch/index/mapper/RankFeaturesFieldMapper.java

```diff
@@ -161,4 +161,9 @@ protected String contentType() {
         return CONTENT_TYPE;
     }
 
+    @Override
+    public Map<String, NamedAnalyzer> indexAnalyzers() {
+        return Collections.singletonMap(name(), Lucene.KEYWORD_ANALYZER);
```

Comment:
should we have an easier path for this common scenario where there is only one analyzer and it is registered with the same name as the field? the mapper in this case only needs to expose which analyzer it is? Having two paths is not optimal, but I think it would help highlighting which mappers are different in that they provide multiple analyzers.